src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java

```diff
@@ -278,7 +279,7 @@ CompletableFuture<Void> flush(boolean isForce) {
   private void createWorkers() {
     // Create thread for checking and scheduling flush job
     ThreadFactory flushThreadFactory =
-        new ThreadFactoryBuilder().setNameFormat("ingest-flush-thread").build();
+        new ThreadFactoryBuilder().setNameFormat("ingest-flush-thread").setDaemon(true).build();
```

Comment:
I have reverted this change. This would have been fine but unneeded since we already shutdown the threadpool when the client is closed. I was having some exceptions in the new benchmark that prevented the shutdown call and I thought we never shutdown the thread pool.